tools/wpt/revlist.py

```diff
@@ -0,0 +1,167 @@
+import argparse
+import calendar
+import logging
+import os
+import time
+from tools.wpt.testfiles import get_git_cmd
+
+here = os.path.dirname(__file__)
+wpt_root = os.path.abspath(os.path.join(here, os.pardir, os.pardir))
+
+logger = logging.getLogger()
+
+DEFAULT_EPOCH = 86400
+DEFAULT_EPOCH_THESHOLD = 600
+DEFAULT_EPOCH_UNTIL = calendar.timegm(time.gmtime())
+DEFAULT_MAX_COUNT = 1
+
+FOR_EACH_REF_MAX_COUNT = 100000
```

Comment:
Inlining this at the only place it's used would be fine I think, it's as easy to change there as here.